pkg/autodetect/main.go

```diff
@@ -221,6 +224,37 @@ func (b *Background) detectKafka(apiList *metav1.APIGroupList) {
 	}
 }
 
+func (b *Background) hasClusterRolePermission() bool {
+	list := &rbac.ClusterRoleBindingList{}
+	opts := []client.ListOption{}
+	err := b.clReader.List(context.Background(), list, opts...)
+	if err != nil {
+		log.WithField("errormsg", err).Debug("Failed to list cluster role bindings")
+		if strings.HasSuffix(err.Error(), "cluster scope") {
```

Comment:
Not quite sure I agree here... I *think* I would return `false` no matter the error. What do you think, @rubenvp8510?

Reply:
That also makes sense

Reply:
Yes, I think it makes more sense to return false here, TBH I don't like to have to check the error message string for validations, but there is no other way.

So we can assume that if there is an error here we don't have permissions or check the error, I would prefer to do the assumption and log the error just in case.